cg/meta/workflow/mip.py

```diff
@@ -182,7 +182,7 @@ def convert_panels(customer: str, default_panels: list[str]) -> list[str]:
                     all_panels.add(extra_panel)
 
         # add OMIM to every panel choice
-        all_panels.add(GenePanelMasterList.OMIM_AUTO)
+        all_panels.add(GenePanelMasterList.OMIM_AUTO.PANELAPP-GREEN)
```

Comment:
I'm not familiar with this syntax? I think what you may want is:
```suggestion
        all_panels.add(GenePanelMasterList.OMIM_AUTO)
        all_panels.add(GenePanelMasterList.PANELAPP-GREEN)
```

Reply:
Yes, that sounds better!